### PR TITLE
Maps in dashboard (map overviews, webmap)

### LINKED ISSUE #500: Documentation page for map overviews
Add new fresh page for informations about map overviews in dashboard.

- Where are map overviews
- When are refreshing map overviews data in map (after push)
- Which data are showing (just in memory layers, vector / raster)
- Define extent of map overviews (qgis extent + push)
- Troubleshooting (Write If something)

### PR DESCRIPTION
**Pull request by alex-cit**
fix #500
Map overviews (webmap) page added to the docs.

### CODE REVIEW

File: src/.vuepress/sidebar/en.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ module.exports = {
                 '/manage/plugin-multi-server-use/',
                 '/manage/plugin/',
                 '/manage/dashboard/',
+                '/manage/webmap/',
```
Comment (MarcelGeo) [inline src/.vuepress/sidebar/en.js:47]: Add map 'map-overviews'

File: src/manage/webmap/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+﻿# Map Overviews
+[[toc]]
+
+**Map overviews** can be displayed in the **Map** tab of a project on <DashboardLink />. You need to be logged in to see the map.
+
+![Mergin Maps dashboard map overviews](./dashboard-webmap.gif "Mergin Maps dashboard map overviews")
+
+:::tip Usage details
+Map Overviews can be used with *Team*, *Academia* and *Non-profit* <MainDomainNameLink id="pricing" desc="subscription plans"/>, as well as in [<CommunityPlatformName />](../../server/) and [<EnterprisePlatformName />](../../server/).
```
Comment (MarcelGeo) [inline src/manage/webmap/index.md:9]: Map overviews are not available in CE and EE now :)

File: src/manage/webmap/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+You can try the Map Overview feature for free during the trial period.
+:::
+
+## Map overviews content
+
+In the map overview, you will see your survey layers or raster layers that are [packaged](../project/#packaging-qgis-project) with the project. Other layers, such as online [background maps](../../gis/settingup_background_map/#background-maps) are not displayed.
+
+The background map that is used for all map overviews is [<MainPlatformName /> vector tile service](../../gis/settingup_background_map/#mergin-maps-vector-tile-service).
+
+The content of the map overview as well as the **Layers** panel are refreshed after every update. This means that you should always see here your current spatial data.
```
Comment (MarcelGeo) [inline src/manage/webmap/index.md:22]: every push to project / synchronization

File: src/manage/webmap/index.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Map Overviews can be used with *Team*, *Academia* and *Non-profit* <MainDomainNameLink id="pricing" desc="subscription plans"/>, as well as in [<CommunityPlatformName />](../../server/) and [<EnterprisePlatformName />](../../server/).
+
+It is not available in *Individual* or *Professional* subscriptions.
+
+You can try the Map Overview feature for free during the trial period.
+:::
+
+## Map overviews content
+
+In the map overview, you will see your survey layers or raster layers that are [packaged](../project/#packaging-qgis-project) with the project. Other layers, such as online [background maps](../../gis/settingup_background_map/#background-maps) are not displayed.
```
Comment (MarcelGeo) [inline src/manage/webmap/index.md:18]: Yeah... postgres and any other network layers are not available.

File: src/manage/webmap/index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+The background map that is used for all map overviews is [<MainPlatformName /> vector tile service](../../gis/settingup_background_map/#mergin-maps-vector-tile-service).
+
+The content of the map overview as well as the **Layers** panel are refreshed after every update. This means that you should always see here your current spatial data.
+
+The **Layers** panel lists all layers displayed on the map. The check button :white_check_mark: controls the visibility of layers.
+
+![Mergin Maps dashboard map overviews](../dashboard/mergin-maps-web-map.jpg "Mergin Maps dashboard map overviews")
+
+### Extent of map overviews
+The extent of map overviews is defined by the [extent set in the QGIS project](../../gis/features/#project-extent) and the extent of the data that are included in the current version of the project.
```
Comment (MarcelGeo) [inline src/manage/webmap/index.md:29]: Under the hood ... there is no combination of extent defined in project and data. If the project extent is not defined, we are trying to get extent from advertised extent:

image

If there is not, layers extents are calculating. This should go to troubleshooting probably :)

**Review by MarcelGeo — CHANGES_REQUESTED** [review]

**Review by MarcelGeo — APPROVED** [review]

File: src/manage/dashboard-maps/index.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+## Troubleshooting
+
+### Map config does not exist
+The **Map** tab of a project on the <DashboardShortLink /> may display this error message:
+`Map config does not exist, please try update the project`
+
+![Mergin Maps webmap Map config does not exist](./webmap-map-config-issue.jpg "Mergin Maps webmap Map config does not exist")
+
+This usually happens when the map was not initiated. All you need to do is to update the project: the map is refreshed after every synchronisation, so your update will activate the map content.
```
Comment (tomasMizera) [inline src/manage/dashboard-maps/index.md:36]: Not sure if people know what it means to "update the project" - how about saying that they need to create a new version or synchronize some change?

**Review by tomasMizera — APPROVED** [review]
It looks all good, thanks for compiling this @alex-cit @MarcelGeo!
I think this is a good starting point, for future we could also add a section about which layers are kept and which are filtered away (the remote ones).

There is just one minor comment from me. Once edited, we can merge 🎉

**Comment by tomasMizera** [comment]
Thanks, let's merge this one :)

**Pull request merged**